openshift/hac-dev
# update slack links on overview page

## Fixes 
https://issues.redhat.com/browse/RHTAPBUGS-290

## Description
The #software-supply-chain-security link we have on RHTAP Overview only takes users to that channel in DevNation Slack if they're already in the DevNation Slack instance. If they've never signed in there before, it doesn't help them get authenticated for the new instance.

Updated copy to include DevNation slack link https://dn.dev/DevNationSlack in accordance with provided UX copy.

## Type of change



## Screen shots / Gifs for design review 


![image](https://github.com/openshift/hac-dev/assets/14068621/c21fb23d-3cb5-4e19-bcdc-1a9bdd2c2450)


## How to test or reproduce?
Join the wait list to see the top banner.

Visit the overview page at any time to see the updated `Contact us` card.


## Browser conformance:
